Assets/Lua/Doom/dsda-data.lua

```diff
@@ -156,6 +157,126 @@ dsda.mobj = dsda.struct_layout()
 	.s32("iden_nums")
 	-- the rest are non-doom
 
+-- sector_t https://github.com/TASEmulators/dsda-doom/blob/5608ee441410ecae10a17ecdbe1940bd4e1a2856/prboom2/src/r_defs.h#L124-L213
+dsda.sector = dsda.struct_layout()
+	.s32  ("iSectorID")
+	.u32  ("flags")
+	.s32  ("floorheight")
+	.s32  ("ceilingheight")
+	.u8   ("soundtraversed")
+	.ptr  ("soundtarget")
+	.array("blockbox", "s32", 4)
+--	.array("bbox", "s32", 4)
+	.s32  ("bbox_top")
+	.s32  ("bbox_bottom")
+	.s32  ("bbox_left")
+	.s32  ("bbox_right")
+	.add  ("soundorg", 60, 8) -- degenmobj_t;
+	.s32  ("validcount")
+	.s32  ("gl_validcount")
+	.ptr  ("thinglist")
+	.s32  ("friction")
+	.s32  ("movefactor")
+	.ptr  ("floordata")
+	.ptr  ("ceilingdata")
+	.ptr  ("lightingdata")
+	.s8   ("stairlock")
+	.s32  ("prevsec")
+	.s32  ("nextsec")
+	.s32  ("heightsec")
+	.s16  ("bottommap")
+	.s16  ("midmapmap")
+	.s16  ("topmap")
+	.s16  ("colormap")
+	.ptr  ("touching_thinglist")
+	.s32  ("linecount")
+	.ptr  ("lines")
+	.s32  ("floorsky")
+	.s32  ("ceilingsky")
+	.s32  ("floor_xoffs")
+	.s32  ("floor_yoffs")
+	.s32  ("ceiling_xoffs")
+	.s32  ("ceiling_yoffs")
+	.s32  ("floorlightsec")
+	.s32  ("ceilinglightsec")
+	.s16  ("floorpic")
+	.s16  ("ceilingpic")
+	.s16  ("lightlevel")
+	.s16  ("special")
+	.s16  ("tag")
+	.s32  ("cachedheight")
+	.s32  ("scaleindex")
+  	-- e6y
```

Comment:
That's just signature of who added it to the code.
https://doomwiki.org/wiki/Andrey_Budko_(entryway)